pypeit/scripts/chk_calibs.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script examines a set of files and indicates which do and
+which do not have sufficient calibs
+"""
+import argparse
+from pypeit import defs
+
+from IPython import embed
+
+def parser(options=None):
+    # TODO: Add argument that specifies the log file
+    parser = argparse.ArgumentParser(description="Script to setup a PypeIt run [v3]")
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('-r', '--root', type=str, default=None,
+                       help='File path+root, e.g. /data/Kast/b ')
+    parser.add_argument('-s', '--spectrograph', default=None, type=str,
+                        help='A valid spectrograph identifier: {0}'.format(
+                                ', '.join(defs.pypeit_spectrographs)))
+    parser.add_argument('-e', '--extension', default='.fits',
```

Comment:
Is there a reason why we don't follow the same parsing rules as pypeit_setup here where we just need to specify the root?

Reply:
We require the instrument